daemon/license.go

```diff
@@ -34,7 +34,7 @@ func CheckLicense(objMgr *ibclient.ObjectManager, licenseType string) (err error
 			}
 		}
 	}
-	err = fmt.Errorf("%s License not available/applied. Apply the license for the grid and try again", GetLicenseName(licenseType))
+	err = fmt.Errorf("%s License not available or User not having sufficient permissions. ", GetLicenseName(licenseType))
```

Comment:
You probably should elaborate what `User` and `permission` the message is referring to.

Reply:
done

Reply:
@yuewko we can't elaborate what permission because we decided not to implement a big logic which we decided will be too much for this small plugin, so trying to address that using documentation.

User ==> Infoblox wapi user

Reply:
@jkraj , yes, just elaborate that it is referring to **WAPI** user and permission.